xml/index.xml

```diff
@@ -7,6 +7,13 @@
 
 <news name="nginx news" link="/" lang="en">
 
+<event date="2024-09-04">
+<para>
+The <link doc="en/index.xml">nginx</link> project has officially moved to
+<link url="https://github.com/nginx/nginx">GitHub</link>.
```

Comment:
Previously, we had a news wrt introducing Trac:
```
<event date="2011-08-03">
<para>
We are happy to announce
<link url="http://trac.nginx.org">trac.nginx.org</link>.
</para>

<para>
You will find the code browser and the bug tracker on this page.
Please note that we would greatly appreciate if you could switch
to this new and hopefully convenient way of submitting bug reports to us,
along with patches to the code.
Here is <link url="http://trac.nginx.org/nginx/wiki">a short instruction</link>.
</para>
```
Now that Trac is retired, a replacement should be mentioned.